### ISSUE TITLE
Can't find .csproj

### ISSUE THREAD

**Issue by AlexanderJohnston** [NecropolisModding/AbraxisToolset#1]
Hi, I can't find the .csproj files for `Assembly-CSharp.Base.mm` and `Assembly-UnityEngine.mm`. It looks like they're meant to be included, and the .sln file will try to load them in VS2017. Is it possible they were ignored from the commit or meant to be sourced from a third party?

It looks like your `.gitignore` has this line:

`# MightyMoose`
`*.mm.*`

Thank you in advance.

**Comment by sshado** [comment]
I went ahead and dropped a message on Patreon to help grab their attention; I'm also having this problem.

**Comment by sshado** [comment]
You can ignore this. @AlexanderJohnston I found out there's a continued version over at https://github.com/Cut2TheChase/AbraxisToolset

**Comment by ZandrXandr** [comment]
Yeah, I'm no longer the person in charge of/developing Abraxis Toolset. Best to go to Chase's version instead, it uses Partiality now, too.
